Automattic/wp-calypso
# A4A: Add Team member table.

This PR implements the Team member table.

Closes  https://github.com/Automattic/automattic-for-agencies-dev/issues/954

<img width="1725" alt="Screenshot 2024-08-21 at 4 25 47 PM" src="https://github.com/user-attachments/assets/17a7e9f0-3d8d-4f95-aafb-e6be9467dbfe">



## Proposed Changes

* Implement Team member table using the Data view component. The data are temporarily hardcoded and will be replaced with actual data once the backend endpoint is ready. Also, action buttons do not work at the moment.

## Testing Instructions

* Use the A4A live link and go to the `/team` page.
* Confirm you can see the Team member table. 

## Pre-merge Checklist
